bin/test_projects_cleanup.rb

```diff
@@ -0,0 +1,78 @@
+example = 'login=rubydev+admin@gooddata.com password=secret server=https://staging2-lcm-prod.intgdc.com ruby delete_projects.rb'
+if ARGV.include?('-h')
+  puts 'Example:'
+  puts example
+  exit 0
+end
+
+require 'gooddata'
+
+def delete_project_by_title(title, projects, days = 14, force = false)
+  dead_line = Time.now - days * 60 * 60 * 24
+  filtered_projects = projects.select do |p|
+    p.title.match(title) && p.created < dead_line
+  end
+  filtered_projects.each do |project|
+    if force
+      puts "Deleting: #{project.pid} - #{project.title} - #{project.created}"
```

Comment:
info: Refers to 'project' more than self (maybe move it to another class?) (FeatureEnvy) (reek)